package.json

```diff
@@ -83,5 +85,24 @@
     "ts-jest": "^26.5.6",
     "typescript": "^4.2.4",
     "wait-on": "^5.3.0"
+  },
+  "bundlewatch": {
+    "trackBranches": [
+      "main"
+    ],
+    "files": [
+      {
+        "path": "./packages/core/dist/index.js",
+        "maxSize": "200B"
+      },
+      {
+        "path": "./packages/core/dist/castor.css",
+        "maxSize": "3kB"
+      },
+      {
+        "path": "./packages/react/dist/index.js",
+        "maxSize": "90B"
+      }
+    ]
```

Comment:
💅🏻 [nitpick]

I'd personally prefer a dedicated `.bundlewatch.json` file for these configs.

@josokinas do you have any preferences?

Reply:
Yes, I would prefer a separate file, however I'd call it `.bundlewatchrc.json`.

Downside is that seems like you then need to specify that file for CLI: https://bundlewatch.io/#/getting-started/using-a-config-file

Reply:
Happy to move the config out to a separate file. But yeah, the only disadvantage is it needs to be specified in the CLI command. The Github Action can be easily set up to take the config file path also.